docs/releasing.md

```diff
@@ -0,0 +1,10 @@
+## Releasing
+
+This package can be published by maintainers of the babel-plugin-transform-i18n repository.
+
+### Stable release
+
+To do this:
+- After merging your changes, run `npm version -f major|minor|patch`. This will create a tagged commit changing the version in `package.json`, and the changes in `CHANGELOG.md`.
+- Run `npm publish`.
+- Run `git push && git push --tags`.
```

Comment:
Can we add an extra line here about creating a release in the github UI that links to PRs of all changes included in the release, and has a description of each change?

Reply:
Sure